app/src/main/java/deakin/gopher/guardian/model/Patient.java

```diff
@@ -0,0 +1,215 @@
+package deakin.gopher.guardian.model;
+
+public class Patient {
```

Comment:
You've accidentally removed the Kotlin version of this file (which recently went through to master) and restored the Java version. Would you be able to intergrate your changes into the Kotin file?

Reply:
Converted Patient to Kotlin